waterwheel/waterwheel.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, nlp: Language, overwrite_ents: bool = True):
             label: iter for iter, label in enumerate([
                 'OCEAN', 'COUNTRY', 'CANADIAN_PROVINCE', 'RIVER',
                 'US_STATE', 'LAKE', 'MOUNTAIN', 'DRAINAGEBASIN', 
-                'WATERCOURSE', 'WATER_BODY'
+                'WATERCOURSE', 'WATER_BODY', 'CHINESE_PROVINCE'
```

Comment:
In case there is any water body (river or lake) which has the same name as a Chinese Province then as per this order, the water body will be given preference to (in the absence of qualifiers of course). If this is okay then this order can be accepted. Otherwise, just need to change the placement of `CHINESE_PROVINCE` in this priority list.

Reply:
Yes, this is probably the right precedence.